src/mgmt/srv_query.c

```diff
@@ -568,3 +568,102 @@ ds_mgmt_dev_set_faulty(uuid_t dev_uuid, Mgmt__DevStateResp *resp)
 
 	return rc;
 }
+
+struct bio_replace_dev_info {
```

Comment:
It's better to avoid using "bio" as prefix for the functions or data structures of mgmt module.

Reply:
same as above